rest-helix/src/main/java/com/github/twitch4j/helix/interceptor/TwitchHelixClientIdInterceptor.java

```diff
@@ -123,6 +79,7 @@ public TwitchHelixClientIdInterceptor(TwitchHelixBuilder twitchHelixBuilder) {
         this.defaultClientId = twitchAPIBuilder.getClientId();
         this.apiRateLimit = twitchAPIBuilder.getApiRateLimit();
         this.defaultAuthToken = twitchHelixBuilder.getDefaultAuthToken();
+        this.rateLimitTracker = new TwitchHelixRateLimitTracker(this);
```

Comment:
Should this be created in `TwitchHelixBuilder` instead, and passed in the constructor to each class that needs access?

Reply:
This makes injecting the meterRegistry a lot easier when we add metric-support to monitor bucket-usage in the future - so this is a good idea.